modules/mysql/main.tf

```diff
@@ -35,7 +29,7 @@ resource "google_sql_database_instance" "default" {
     activation_policy           = "${var.activation_policy}"
     authorized_gae_applications = ["${var.authorized_gae_applications}"]
     backup_configuration        = ["${var.backup_configuration}"]
-    ip_configuration            = "${local.ip_configurations["${local.ip_configuration_enabled ? "enabled" : "disabled"}"]}"
+    ip_configuration            = ["${var.ip_configuration}"]
```

Comment:
By this change, `make test_integration` fails, because this still recognizes empty `ip_configuration` even if `ip_configuration` is unspecified.

Current behavior:
```
         settings.#:                                   "" => "1"
         settings.0.activation_policy:                 "" => "ALWAYS"
         settings.0.availability_type:                 "" => "<computed>"
         settings.0.backup_configuration.#:            "" => "1"
         settings.0.backup_configuration.0.start_time: "" => "<computed>"
         settings.0.crash_safe_replication:            "" => "<computed>"
         settings.0.disk_autoresize:                   "" => "true"
         settings.0.disk_size:                         "" => "10"
         settings.0.disk_type:                         "" => "PD_SSD"
         settings.0.location_preference.#:             "" => "1"
         settings.0.location_preference.0.zone:        "" => "us-central1-c"
         settings.0.maintenance_window.#:              "" => "1"
         settings.0.maintenance_window.0.day:          "" => "1"
         settings.0.maintenance_window.0.hour:         "" => "23"
         settings.0.maintenance_window.0.update_track: "" => "canary"
         settings.0.pricing_plan:                      "" => "PER_USE"
         settings.0.replication_type:                  "" => "SYNCHRONOUS"
         settings.0.tier:                              "" => "db-n1-standard-1"
         settings.0.version:                           "" => "<computed>"
```

Your changes:
```
         settings.#:                                   "" => "1"
         settings.0.activation_policy:                 "" => "ALWAYS"
         settings.0.availability_type:                 "" => "<computed>"
         settings.0.backup_configuration.#:            "" => "1"
         settings.0.backup_configuration.0.start_time: "" => "<computed>"
         settings.0.crash_safe_replication:            "" => "<computed>"
         settings.0.disk_autoresize:                   "" => "true"
         settings.0.disk_size:                         "" => "10"
         settings.0.disk_type:                         "" => "PD_SSD"
         settings.0.ip_configuration.#:                "" => "1" # <- PTAL
         settings.0.ip_configuration.0.ipv4_enabled:   "" => "<computed>"
         settings.0.location_preference.#:             "" => "1"
         settings.0.location_preference.0.zone:        "" => "us-central1-c"
         settings.0.maintenance_window.#:              "" => "1"
         settings.0.maintenance_window.0.day:          "" => "1"
         settings.0.maintenance_window.0.hour:         "" => "23"
         settings.0.maintenance_window.0.update_track: "" => "canary"
         settings.0.pricing_plan:                      "" => "PER_USE"
         settings.0.replication_type:                  "" => "SYNCHRONOUS"
         settings.0.tier:                              "" => "db-n1-standard-1"
         settings.0.version:                           "" => "<computed>"
```

I would like to revert this changes.

Reply:
Done. The new change restores the old behavior.

```
  connection_name:                              "" => "<computed>"
         database_version:                             "" => "POSTGRES_9_6"
         first_ip_address:                             "" => "<computed>"
         ip_address.#:                                 "" => "<computed>"
         master_instance_name:                         "" => "<computed>"
         name:                                         "" => "tf-pg-simple-x"
         private_ip_address:                           "" => "<computed>"
         project:                                      "" => "mmontan-cloud"
         public_ip_address:                            "" => "<computed>"
         region:                                       "" => "us-central1"
         replica_configuration.#:                      "" => "<computed>"
         self_link:                                    "" => "<computed>"
         server_ca_cert.#:                             "" => "<computed>"
         service_account_email_address:                "" => "<computed>"
         settings.#:                                   "" => "1"
         settings.0.activation_policy:                 "" => "ALWAYS"
         settings.0.availability_type:                 "" => "ZONAL"
         settings.0.backup_configuration.#:            "" => "1"
         settings.0.backup_configuration.0.start_time: "" => "<computed>"
         settings.0.crash_safe_replication:            "" => "<computed>"
         settings.0.disk_autoresize:                   "" => "true"
         settings.0.disk_size:                         "" => "10"
         settings.0.disk_type:                         "" => "PD_SSD"
         settings.0.location_preference.#:             "" => "1"
         settings.0.location_preference.0.zone:        "" => "us-central1-c"
         settings.0.maintenance_window.#:              "" => "1"
         settings.0.maintenance_window.0.day:          "" => "1"
         settings.0.maintenance_window.0.hour:         "" => "23"
         settings.0.maintenance_window.0.update_track: "" => "canary"
         settings.0.pricing_plan:                      "" => "PER_USE"
         settings.0.replication_type:                  "" => "SYNCHRONOUS"
         settings.0.tier:                              "" => "db-f1-micro"
         settings.0.version:                           "" => "<computed>"
```